connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaConfigBackingStore.java

```diff
@@ -179,7 +179,7 @@ public static String COMMIT_TASKS_KEY(String connectorName) {
     // converter/serializer changes causing keys to change. We need to absolutely ensure that the keys remain precisely
     // the same.
     public static final Schema CONNECTOR_CONFIGURATION_V0 = SchemaBuilder.struct()
-            .field("properties", SchemaBuilder.map(Schema.STRING_SCHEMA, Schema.OPTIONAL_STRING_SCHEMA))
+            .field("properties", SchemaBuilder.map(Schema.STRING_SCHEMA, Schema.OPTIONAL_STRING_SCHEMA).build())
```

Comment:
Seems kind of error-prone to have `SchemaBuilder` extend `Schema` since we can't catch cases like this with the type system. Perhaps we should just make `field()` and other methods which accept a schema argument work in `SchemaBuilder` (e.g. by calling build() internally). Maybe the easy way to do it is adding this method to `SchemaBuilder`:

```java
public SchemaBuilder field(String fieldName, SchemaBuilder fieldSchemaBuilder) {
    return field(fieldName, fieldSchemaBuilder.build());
}
```